src/Page/GetHelp.elm

```diff
@@ -15,9 +17,9 @@ view =
             [ h1 [ css [ pageHeadingStyle ] ] [ text (t GetHelpTitle) ]
             ]
         , div [ css [ columnStyle ] ]
-            [ card (t GetHelpSection1Title) (t GetHelpSection1Link) (t SeaSurvivorForumHref)
-            , card (t GetHelpSection2Title) (t GetHelpSection2Link) (t SeaMapFinancialSupportHref)
-            , card (t GetHelpSection3Title) (t GetHelpSection3Link) (t SeaOrganisationsResourceHref)
+            [ card (t GetHelpSection1Title) (t GetHelpSection1Quote1) (t GetHelpSection1Description) (t GetHelpSection1Link) (t SeaSurvivorForumHref)
```

Comment:
Feels weird having an href Key here.

What about `GetHelpSection2Link Key` for all of our links that have Hrefs so it is impossible to render them without including an associate Key?